doc/StereoStreaming.md

```diff
@@ -56,18 +67,12 @@ Example of a stereo 3D client application using the blocking Stream API:
     /** ...synchronize with other render clients (network barrier)... */
     }
 
-Example of a stereo 3D client application using the asynchronous Stream API:
+Example of a stereo 3D client application using the asynchronous operations:
 
     deflect::Stream stream( ... );
 
     /** ...synchronize start with other render clients (network barrier)... */
 
-    delfect::Stream::Future leftFuture, rightFuture;
-    leftFuture = deflect::qt::make_ready_future<bool>( true );
-    rightFuture = deflect::qt::make_ready_future<bool>( true );
-
-    ImageData leftData, rightData; // must remain valid until sending completes
```

Comment:
why remove this? I think it is important to show the correct usage of the api. Without those variables the renderLoop below does not make sense because the "auto future" goes out of scope.

Furthermore there should be a third future for the deflectStream->finishFrame();